src/Action/Bulk/ToggleAction.php

```diff
@@ -0,0 +1,68 @@
+<?php
+namespace Crud\Action\Bulk;
+
+use Cake\Controller\Controller;
+use Cake\Database\Expression\QueryExpression;
+use Cake\ORM\Query;
+use Crud\Error\Exception\ActionNotConfiguredException;
+
+/**
+ * Handles Bulk 'Toggle' Crud actions
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ */
+class ToggleAction extends BaseAction
+{
+    /**
+     * Constructor
+     *
+     * @param \Cake\Controller\Controller $Controller Controller instance
+     * @param array $config Default settings
+     * @return void
+     */
+    public function __construct(Controller $Controller, $config = [])
+    {
+        $this->_defaultConfig['actionName'] = 'Toggle';
+        $this->_defaultConfig['messages'] = [
+            'success' => [
+                'text' => 'Value toggled successfully'
+            ],
+            'error' => [
+                'text' => 'Could not toggle value'
+            ]
+        ];
+        return parent::__construct($Controller, $config);
+    }
+
+    /**
+     * Handle a bulk event
+     *
+     * @return \Cake\Network\Response
+     */
+    protected function _handle()
+    {
+        $field = $this->config('field');
+        if (empty($field)) {
+            throw new ActionNotConfiguredException('No field value specified');
+        }
+
+        return parent::_handle();
+    }
+
+    /**
+     * Handle a bulk toggle
+     *
+     * @param \Cake\ORM\Query $query The query to act upon
+     * @return bool
+     */
+    protected function _bulk(Query $query = null)
```

Comment:
Yeah, what @ADmad said. If someone wants to be *more* clear about their application implementation, there is also `SetValueAction`.